coreModulePackages/promise/index.js

```diff
@@ -16,6 +16,7 @@
 // with Webpack 2+. We need `require('promise-polyfill').default` for running the tests
 // and `require('promise-polyfill')` for building Turbine.
 module.exports =
-  window.Promise ||
+  (typeof window !== 'undefined' && window.Promise) ||
+  (typeof global !== 'undefined' && global.Promise) ||
```

Comment:
Maybe just `(window || global).Promise ||` instead? I've also seen this format:

```
module.exports = (function(root) {
  root.Promise ||
  require('promise-polyfill').default ||
  require('promise-polyfill');	  require('promise-polyfill');
})(window || global);
```

Whatever you want to do here works for me though.

Reply:
This is what I get in node:

```
window || global
Thrown:
ReferenceError: window is not defined
```